src/components/MessageBar/MessageBar.md

```diff
@@ -1,47 +1,54 @@
-Renders a component that displays relevant status information. You can use a `MessageBar` to tell the user about a situation and optionally provide actions for them to take.
-
 ### Notes for use
 
-Text that is too long for the context will wrap to two or more lines but you should aim for short strings. Actions will grow from the right but should also take up as little horizontal space as possible.
+Text that is too long for the context will wrap to two or more lines, but you should aim for short strings. Actions will grow from the right but should also take up as little horizontal space as possible.
 
 ### Examples
 
-Default info type:
+Informational:
 
-```js { "props": { "data-example": "info" } }
-<MessageBar ariaLabel="A sample notice informing you about something">
-  Sample notice, something you should know about.
+```js { "props": { "data-example": "basic" } }
+<MessageBar ariaLabel="A sample informational message">
+  Something you should know about.
 </MessageBar>
 ```
 
 Warning:
 
-```js { "props": { "data-example": "warning" } }
+```js { "props": { "data-example": "with type" } }
 const { MessageBarType } = require('.');
 
-<MessageBar ariaLabel="A sample notice warning you about something" type={MessageBarType.WARNING}>
+<MessageBar ariaLabel="A sample warning message" type={MessageBarType.WARNING}>
   Watch out, something unexpected might happen.
 </MessageBar>
 ```
 
-Error:
+Warning with an icon action:
 
-```js { "props": { "data-example": "error" } }
+```js { "props": { "data-example": "with icon action" } }
+const { IconSize } = require('../Icon');
+const { TextSize } = require('../Text');
 const { MessageBarType } = require('.');
 
-<MessageBar ariaLabel="A sample notice about an error" type={MessageBarType.ERROR}>
```

Comment:
i had updated the MessageBar examples to explicitly show an info, warning and error example, did you not like that?

Reply:
Sorry, I had a pretty epic merge conflict because I had forgotten to push yesterday, and you had also pushed a bunch of changes.

Reply:
no prob. we can continue cleaning up examples later on